OracleDatabase/SingleInstance/FAQ.md

```diff
@@ -110,3 +114,65 @@ Refer to the [module documentation](https://python-oracledb.readthedocs.io/en/la
 ## ORA-01157: cannot identify/lock data file
 
 This error occurs when the database cannot find a data file (used for tablespaces) that was previously present. This is most likely because the data file has been located outside the volume in a previous container and was hence not persisted. Ensure that when you add tablespaces and/or data files that they are located within the volume location, i.e. $ORACLE_BASE/oradata/$ORACLE_SID, (e.g. `/opt/oracle/oradata/XE`).
+
+## Running Oracle Database 23c Free on Apple based ARM M1 chip
+
+### Setup Database
+Below are the steps to run Oracle Database 23c Free on Apple ARM M1 machine
+
+1. Install [Podman Lima](https://github.com/lima-vm/lima) on Mac
+
+    ```brew install podman lima```
+
+2. Start a lima instance (vm)
+
+    ```limactl start --name podman-amd64  --set='.arch = "x86_64" | .memory = "10GiB"'  template://podman```
+
+    Wait for the VM to boot up. Above command may error out if the VM takes a while to boot up. Ultimately it comes up fine.
+
+3. Set the address or hostname of the lima container runtime
+
+    ```export CONTAINER_HOST=unix://Users/$USER/.lima/podman-amd64/sock/podman.sock```
+
+4. Pull the oracle database 23c free image
+
+    ```podman pull container-registry.oracle.com/database/free:latest```
+
+5. Run the container with the pulled image
+
+    ```podman run -e ORACLE_PWD=<password> -d -P container-registry.oracle.com/database/free:latest```
+
+6. Check whether the database came up healthy.
+
+```shell
+    $ podman ps -a
+        CONTAINER ID  IMAGE                                               COMMAND               CREATED        STATUS                  PORTS                    NAMES 
+        825dcbd3e822  container-registry.oracle.com/database/free:latest  /bin/sh -c exec $...  4 minutes ago  Up 4 minutes (healthy)  0.0.0.0:42439->1521/tcp  hopeful_yalow
+```
+
+### Connect
+1. Install sqlplus
+
+   ```cd $HOME/Downloads```
+
+   ```curl -O https://download.oracle.com/otn_software/mac/instantclient/198000/instantclient-basic-macos.x64-19.8.0.0.0dbru.dmg```
+
+   ```curl -O https://download.oracle.com/otn_software/mac/instantclient/198000/instantclient-sqlplus-macos.x64-19.8.0.0.0dbru.dmg```
+
+   ```hdiutil mount instantclient-basic-macos.x64-19.8.0.0.0dbru.dmg```
+
+   ```hdiutil mount instantclient-sqlplus-macos.x64-19.8.0.0.0dbru.dmg```
+
+   ```/Volumes/instantclient-basic-macos.x64-19.8.0.0.0dbru/install_ic.sh```
+
+   ```hdiutil unmount /Volumes/instantclient-basic-macos.x64-19.8.0.0.0dbru```
+
+   ```hdiutil unmount /Volumes/instantclient-sqlplus-macos.x64-19.8.0.0.0dbru```
+
+2. Test the connection:
+
+    ```$HOME/Downloads/instantclient_19_8/sqlplus sys/<ORACLE_PWD>@localhost:<PORT>/FREE as sysdba```
```

Comment:
- For security reasons, don't include the password in the command line.  SQL*Plus will prompt for it.

- I don't know the Oracle DB Dockerfile standard, but for everyday use it is more useful to show connecting to the PDB than to the CDB.

- Some users may prefer sqlcl so you could offer those steps as an alternative, depending how much complexity you want to add in this FAQ.

Reply:
Fixed. Thanks